internal/commands/push.go

```diff
@@ -43,327 +35,224 @@ const ( // Docker specific annotations and values
 	DockerTypeApp = "app"
 )
 
-type pushOptions struct {
-	tag          string
-	platforms    []string
-	allPlatforms bool
-}
-
 func pushCmd(dockerCli command.Cli) *cobra.Command {
-	var opts pushOptions
 	cmd := &cobra.Command{
-		Use:     "push [APP_NAME] --tag TARGET_REFERENCE [OPTIONS]",
+		Use:     "push APP_IMAGE[:TAG]",
 		Short:   "Push an application package to a registry",
-		Example: `$ docker app push myapp --tag myrepo/myapp:mytag`,
-		Args:    cli.RequiresMaxArgs(1),
-		PreRunE: func(cmd *cobra.Command, args []string) error {
-			return checkFlags(cmd.Flags(), opts)
-		},
+		Long:    "Push an application, including all the service images and the invocation image to a registry",
```

Comment:
Push an application image? Just to explicitly say that we can only push built applications?